packages/integration-sdk-cli/src/memgraph/memgraphGraphStore.ts

```diff
@@ -0,0 +1,181 @@
+import { Entity, Relationship } from '@jupiterone/integration-sdk-core';
+import {
+  sanitizeValue,
+  buildPropertyParameters,
+  sanitizePropertyName,
+  getFromTypeLabel,
+  getToTypeLabel,
+} from './memgraphUtilities';
+
+import * as memgraph from 'neo4j-driver';
```

Comment:
Does Memgraph use the same Javascript driver as Neo4j?  If so, would we be able to share some of the same codebase with our existing Neo4j graph storage?

Reply:
Yes, Memgraph is compatible with Neo4j and uses the same JS driver so feel free to share some of the codebase. I separated everything to preserve the integrity of your code, but feel free to make the necessary changes